aymenfurter/smartrag
# Fix: Add Missing Queues to Infrastructure Script

### Summary
This PR addresses the issue where the container fails to start due to missing Azure storage queues, causing a `QueueNotFoundError`. The root cause was the absence of the necessary queues in the infrastructure script.

### Details:
- Added the required Azure storage queues for:
  - **Pending file uploads**
  - **Pending index**
  
These queues are now created during the infrastructure setup process to prevent the `QueueNotFoundError` that was being encountered.

### Steps to Test:
1. Run `azd up` to deploy the infrastructure.
2. Verify that the two queues (`ingestion`, `indexing`) are created in the Azure Storage account.
3. Ensure the container starts successfully without any queue-related errors.

### Linked Issue:
- [Issue #12](https://github.com/aymenfurter/smartrag/issues/12) - Container is throwing error while trying to start.
